website/docusaurus.config.js

```diff
@@ -164,6 +164,7 @@ const config = {
       prism: {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
+        // additionalLanguages: ['graphql'],
```

Comment:
To remove if not needed please

Reply:
does the current theme work with graphql syntax? added this as a comment since i wasn't sure